script/DeployMainnet.s.sol

```diff
@@ -71,13 +71,13 @@ contract DeployMainnet is DeployBase {
         config.keyRemovalCharge = 0.05 ether;
         config
             .elRewardsStealingReporter = 0xC52fC3081123073078698F1EAc2f1Dc7Bd71880f; // CSM Committee MS
-        // EarlyAdoption
+        // VettedGate
         config
-            .earlyAdoptionTreeRoot = 0x359e02c5c065c682839661c9bdfaf38db472629bf5f7a7e8f0261b31dc9332c2; // See the first value in artifacts/mainnet/early-adoption/merkle-tree.json
-        config.earlyAdoptionBondCurve = new uint256[](2);
+            .vettedGateTreeRoot = 0x359e02c5c065c682839661c9bdfaf38db472629bf5f7a7e8f0261b31dc9332c2; // See the first value in artifacts/mainnet/early-adoption/merkle-tree.json
```

Comment:
We need to update the root before deployment to avoid double claming

Reply:
Add a comment pls

Reply:
there is room for frontrunning anyway. if somebody decides to claim EA before CSM plug-in the gate for v2

Reply:
We accept this risk as we already have duplicates (multiple addresses owned by a single person/entity) in the current list